test/cli/rbi-gen-single/rbi-gen-single.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+test_path="$(dirname "$0")"
+
+sorbet="$PWD/main/sorbet"
+
+rbis="$(mktemp -d)"
+trap "rm -rf $rbis" EXIT
+
+echo "-- sanity-checking package with --stripe-packages"
+"$sorbet" --silence-dev-message \
+  --stripe-packages \
+  --dump-package-info="$rbis/package-info.json" \
+  "$test_path"
+
+# Generate rbis for each package
+find . -name __package.rb | sort | while read -r package; do
+  name="$(awk '/class/ {print $2}' "$package")"
```

Comment:
This is totally non-blocking, but this script is doing very similar things to the existing single-package typechecking tests; it would be nice if they were merged somehow.  I don't know how to make that happen, exactly, because this script is checking for different things than the existing test.  But it wouldn't surprise me if we wind up needing more tests of this flavor.

Reply:
I looked into merging it in, but wanted to get this PR merged with some testing present. It would be totally reasonable to rework this test into existing tests, but I'd rather not block this PR on that.